tap/api/api.go

```diff
@@ -7,13 +7,13 @@ import (
 	"errors"
 	"fmt"
 	"io/ioutil"
+	"log"
```

Comment:
why are we using log and not the logger in shared?

Reply:
@RoyUP9 logging in protocol extensions is little bit tricky. Because there could be a lot of false negatives in protocol extensions. Since they blindly try to parse the TCP stream in their own protocol specification. In fact, logging should be minimized in the protocol extensions.

Introducing more dependencies to protocol extensions is also not great. (`go-logging`) Because protocol extensions meant to be small shared objects that do their jobs and nothing more. They are compiled separately and adding more dependencies increases their binary size. Hence the load time increases and the runtime slows down.

`github.com/up9inc/mizu/shared` is bloated by irrelevant code. Once something changes in that module, all extensions should be updated as well and since the code in `github.com/up9inc/mizu/shared` constantly, it's better to not depend on that module.

We'll eventually remove almost all logging from the protocol extensions but it's better to remove it gradually.

Reply:
@mertyildiran 
We want to have those logs because it doesn't look right to comment out a lot of logs.
Also, there is no real reason to take the shared logger into extensions (anyway it is really one repo and they are taking it locally using the "replace"). 
If you can do it we will merge it and maybe think if there should be a better solution.

Reply:
@mertyildiran 

Ok, we prefer it like this but just need to remove another 2 logs in api.go (line 254 and 266), those are also not relevant like others so we will loose it totally and not keeping some of them